### PR TITLE
Annotate available OS types for Clock and newer time-based inits

### PR DESCRIPTION
**Pull request by aim2120**

### CODE REVIEW

File: Sources/Afluent/SequenceOperators/DelaySequence.swift [hunk 1]
```diff
@@ -8,6 +8,7 @@
 import Foundation
 
 extension AsyncSequences {
+    @available(iOS 16, *)
```
Comment (Tyler-Keith-Thompson) [inline Sources/Afluent/SequenceOperators/DelaySequence.swift:11]: Can you please add all other OS requirements? macOS, tvOS, watchOS, etc.

Comment (aim2120) [reply Sources/Afluent/SequenceOperators/DelaySequence.swift]: Yep! On it.

Comment (aim2120) [reply Sources/Afluent/SequenceOperators/DelaySequence.swift]: Pushed

**Comment by Tyler-Keith-Thompson** [comment]
Great MR! Thanks! Surprised CI didn't force this.

**Pull request merged**